### ISSUE TITLE
Update assets aliases

### ISSUE THREAD

**Issue by chervand** [yiisoft/yii2#15088]
Since switched to `asset-packagist.org`, `yii\base\Application::setVendorPath()` `bower` and `npm` paths should be updated `bower-asset` and `npm-asset` respectively.

### What steps will reproduce the problem?

Using path aliases such as `@bower` or `@npm`. For example, `yiisoft/yii2-bootstrap` uses `@bower` alias for assets.

### What is the expected result?

 `@bower` and `@npm` are aliases for `@vendor/bower-asset` and `@vendor/npm-asset` respectively.

### What do you get instead?

Wrong paths for `bower` and `npm` assets while using its path aliases.

### Additional info

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.13
| PHP version      | 7.0.22
| Operating system | ubuntu0.16.04.1

**Comment by samdark** [comment]
That should be already handled in application templates... @SilverFire could you take a look?

**Comment by chervand** [comment]
The only way to achieve it in applications, I believe, is to override such aliases. Are You going to say every existing app should include it to upgrade?

**Comment by samdark** [comment]
If you're using asset-packagist — yes, you have to override aliases. If you're using existing apps and not switching to asset-packagist and have fxp composer plugin then it should work as it worked before.

**Comment by cebe** [comment]
you have to configure aliases in application config when using asset-packagist: https://github.com/yiisoft/yii2-app-basic/blob/2.0.13/config/web.php#L10-L13

**Comment by cebe** [comment]
For 2.0.x the framework defaults will stay. we can change them in 2.1 though.

**Comment by chervand** [comment]
Yes, this is what I'm talking about. 2.0.13 uses asset-packagist, but aliases still points to not existing asset paths. This makes apps using such aliases,  `yiisoft/yii2-apidoc` and similar libs not usable without symlinks in `vendor`.

**Comment by cebe** [comment]
the usage of asset-packagist vs. composer-asset-plugin depends on the application template. Both templates, yii2-app-basic and yii2-app-advanced define the aliases correctly for asset-packagist, so there is nothing to adjust on the 2.0.x branch.

**Comment by schmunk42** [comment]
> Since switched to `asset-packagist.org`

How did you switch, did you also uninstall or disable fxp/cap?

**Comment by chervand** [comment]
@schmunk42 I've uninstalled `fxp` plugin and made `composer update`, seems like `yii2` already has it [configured](https://github.com/yiisoft/yii2/blob/2.0.13/composer.json#L88-L93).

**Comment by schmunk42** [comment]
The section you've linked won't have any affect on your project. You have to add repositories manually to your root composer.json (see https://getcomposer.org/doc/04-schema.md#repositories).

Could you show the output of:
- `composer global show`
- `composer show`
- `composer update -vvv` as a gist

**Comment by chervand** [comment]
@schmunk42 ok, sorry for misunderstanding. In case you're trying to help me, I've already resolved my problems. I've added `asset-packagist` to my root `composer.json` as well and my previous comment isn't clear enough, sorry for that.

This issue was basically a suggestion to update core paths which are kind of confusing. The main problem was inability to generate project docs with `yiisoft/yii2-apidoc` which doesn't respects project config and uses core asset paths in its current state. 

As was mentioned above, default paths are going to be updated in a future release, so I'm going to leave symlinks in vendor for `apidoc` by now.
